Formula/gomplate.rb

```diff
@@ -0,0 +1,33 @@
+class Gomplate < Formula
+  desc "Command-line Golang template processor"
+  homepage "https://gomplate.hairyhenderson.ca/"
+  url "https://github.com/hairyhenderson/gomplate/archive/v1.9.1.tar.gz"
+  sha256 "4d642bcb6d72b793dadaa622b4a2f42efea1f2425f428798f1feacdbf158f812"
+  head "https://github.com/hairyhenderson/gomplate.git"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+  depends_on "upx" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    (buildpath/"src/github.com/hairyhenderson/gomplate").install buildpath.children
+    cd "src/github.com/hairyhenderson/gomplate" do
+      system "make", "compress", "VERSION=#{version}"
+      bin.install "bin/gomplate-slim" => "gomplate"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    ver = shell_output "#{bin/"gomplate"} --version"
```

Comment:
Actually, that is to test that we compiled so that the version number is set. I'll add a comment to that effect.